packages/cardpay-cli/scheduled-payment/cancel.ts

```diff
@@ -0,0 +1,49 @@
+import { Argv } from 'yargs';
+import { getConstant, getSDK } from '@cardstack/cardpay-sdk';
+import { getEthereumClients, getConnectionType, NETWORK_OPTION_ANY } from '../utils';
+import { Arguments, CommandModule } from 'yargs';
+
+export default {
+  command: 'cancel <safeAddress> <moduleAddress> <gasTokenAddress> <spHash>',
+  describe: 'Cancel a scheduled payment',
+  builder(yargs: Argv) {
+    return yargs
+      .positional('safeAddress', {
+        type: 'string',
+        description: 'The address of the safe that will fund the scheduled payment',
```

Comment:
Probably the description should be that it will fund the removal of scheduled payment